gcloud/resource_manager/project.py

```diff
@@ -0,0 +1,323 @@
+            >>> print project
+            <project: Purple Spaceship (purple-spaceship-123)>
+
+        See
+        https://cloud.google.com/resource-manager/reference/rest/v1beta1/projects/get
+
+        :type client: :class:`gcloud.resource_manager.client.Client` or None
+        :param client: the client to use.  If not passed, falls back to
+                       the ``client`` attribute.
+        """
+        client = self._require_client(client=client)
+
+        # We assume the project exists. If it doesn't it will raise a NotFound
+        # exception.
+        resp = client.connection.api_request(method='GET', path=self.path)
+        self.set_properties_from_api_repr(resource=resp)
+
+    def update(self, client=None):
+        """API call:  update the project via a ``PUT`` request.
+
+        Example::
+
+            >>> from gcloud import resource_manager
+            >>> client = resource_manager.Client()
+            >>> project = client.get_project('purple-spaceship-123')
+            >>> project.name = 'New Purple Spaceship'
+            >>> project.labels['environment'] = 'prod'
+            >>> project.update()
+
+        See
+        https://cloud.google.com/resource-manager/reference/rest/v1beta1/projects/update
+
+        :type client: :class:`gcloud.resource_manager.client.Client` or None
+        :param client: the client to use.  If not passed, falls back to
+                       the ``client`` attribute.
+        """
+        client = self._require_client(client=client)
+
+        data = {'name': self.name, 'labels': self.labels}
+        resp = client.connection.api_request(method='PUT', path=self.path,
+                                             data=data)
+        self.set_properties_from_api_repr(resp)
+
+    def exists(self, client=None):
+        """API call:  test the existence of a project via a ``GET`` request.
+
+        Example::
+
+            >>> from gcloud import resource_manager
+            >>> client = resource_manager.Client()
+            >>> project = client.project('purple-spaceship-456')
+            >>> project.exists()
+            False
+
+        You can also use the
+        :func:`gcloud.resource_manager.client.Client.get_project`
+        method to check whether a project exists, as it will return ``None``
+        if the project doesn't exist::
+
+            >>> from gcloud import resource_manager
+            >>> client = resource_manager.Client()
+            >>> print client.get_project('purple-spaceship-456')
+            None
+
+        See
+        https://cloud.google.com/pubsub/reference/rest/v1beta2/projects/projects/get
+
+        :type client: :class:`gcloud.resource_manager.client.Client` or None
+        :param client: the client to use.  If not passed, falls back to
+                       the ``client`` attribute.
+        """
+        client = self._require_client(client=client)
+
+        try:
+            client.connection.api_request(method='GET', path=self.path)
```

Comment:
Yea -- not available on this API (https://cloud.google.com/resource-manager/reference/rest/v1beta1/projects/get)